tests/benchdnn/conv/cfg.cpp

```diff
@@ -45,7 +45,7 @@ const _dt_conf_t conf_f16 = {
         {dnnl_f16, -int_max_exact_half, int_max_exact_half, -6, 6, 0, 1, 1.0,
                 0.},
         {dnnl_f16, -int_max_exact_half, int_max_exact_half, -4, 4, 0, 1, .25,
-                0.},
+                1e-3},
```

Comment:
Hi @fadara01.
Despite reference does its computations in f32, it doesn't affect the output precision.
Benchdnn f32 inputs are converted to f16 and exact f16 values are used for reference as f32 values.
Reference uses f32 accumulator and down-converts the result to a requested precision.
Thus, if the library accumulates in f32, the output should be the same (if no post-ops are provided) because of the special filling used for convolutions.

Could you double-check, please, what data type is used for accumulation inside your specific problem? Thank you.

Reply:
Hi @dzarukin 
You are right that convolution works (accumulation is in fp32), i.e. if you run: 
`./tests/benchdnn/benchdnn --conv --skip-impl=ref --allow-enum-tags-only=false --cfg=f16 mb1ic128ih52oc256oh52kh3ph1n"yolo.inf.fp16.ov.mb1*3"`
the test passes.

Our problem here is when there are post-ops (relu specifically). The post-op (say relu) happens in fp32 (since both `alpha` and `src` are fp32): `FP32(FP16(alpha * src))`  which is different from what actually happens in ACL: `FP32(FP16(alpha) * FP16(src))`

Reply:
oneDNN programming model specifies an application of eltwise post-op in f32 regardless of data types of inputs and outputs. If this is not something that happening, let's talk about it, but, in general, I'd expect that all providers are complaint with the rule of fp32 post-ops. Thanks.

Reply:
This is not what is happening in this case. The output of the convolution in ACL is in f16 and gets directly fed into the post-op, so the post-op in ACL is executed in f16 (it calls the [f16 SVE kernels](https://github.com/ARM-software/ComputeLibrary/blob/main/src/cpu/kernels/activation/generic/sve/fp16.cpp#:~:text=tmp%20%3D%20svadd_f16_z(pg%2C%20svmul_f16_z(pg%2C%20svmin_f16_z(pg%2C%20vin%2C%20const_0)%2C%20va)%2C%20svmax_f16_z(pg%2C%20vin%2C%20const_0))%3B)).
I'll submit another PR to disable ACL post-ops in f16 mode until the issue is fixed.